polyglot/piranha/tests/tests.py

```diff
@@ -0,0 +1,23 @@
+from polyglot_piranha import run_piranha_cli
+from os.path import join, basename
+from os import listdir
+
+
+def test_piranha_rewrite():
```

Comment:
Do these run without `import unittest` and being part of a Python `unittest.TestCase`? I also don't see anything in `pyproject.toml` specifying to run this file... is it by convention from the name? Have we verified that this fails the build/CI if we include a failing test?

Reply:
Oh boy! I wondered the same for a lot of time staring at the example repos.

So `pytest` uses name based heuristics to decide which is the test method. [Documentation](https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#conventions-for-python-test-discovery) 

I have not tried explicitly the failing scenario, i will try that and report here.

Reply:
yayy! the test fails https://github.com/uber/piranha/runs/8163737912?check_suite_focus=true 
I added a test that asserts `19 == 20`.

Reply:
Oh also, i invoke `pytest` in the Polyglot Piranha's CI file.